packages/manager/modules/pci/src/projects/project/instances/add/add.constants.js

```diff
@@ -18,6 +18,9 @@ export const INSTANCE_READ_MORE_GUIDE = {
       'https://docs.ovh.com/gb/en/public-cloud/attaching-pci-floating-ip-to-instance/',
   },
 };
+export const INSTANCE_PRICES_LINK =
+  'https://www.ovhcloud.com/fr/public-cloud/prices/#397';
+
```

Comment:
May be you need update links for other subsidiaries as well and display based on subsidiary.

Reply:
The current spec indicate only this link.
I'll check with the PO.

Reply:
@sachinrameshn 
Prices links updated.